pegasus/sites.v3/code.org/public/css/homepage.css

```diff
@@ -110,13 +110,31 @@ body.modal-open .supreme-container{
   text-align: center
 }
 
+#homepage #actions a {
+  text-decoration: none;
+}
+
+#homepage #actions #title {
+  font-family: 'Gotham 5r', sans-serif;
+  font-size: 26px;
+  color: white;
+  line-height: 1.2em;
+}
+
+#homepage #actions #subtitle {
+  font-family: 'Gotham 4r', sans-serif;
+  font-size: 16px;
+  color: white;
+  margin: 15px 0;
+}
+
 #homepage #actions #playimage {
-  cursor: pointer
+  cursor: pointer;
 }
 
 #homepage #actions #playtext {
-  font-size: 20px; 
-  cursor: pointer
+  font-size: 16px; 
```

Comment:
Really? I tend to go with delete what we don't use and add it back in when we use it again. I think having something like playtext large and playtext small is unnecessary.